modules/social_features/social_group/src/SocialGroupHelperService.php

```diff
@@ -190,6 +190,8 @@ public function getGroupFromEntity(array $entity, bool $read_cache = TRUE): ?int
 
   /**
    * {@inheritdoc}
+   *
+   * @todo Deprecate this in 11.x and remove.
```

Comment:
@Kingdutch @nechai This comment is confusing (since we already on 12.x)
Is it still relevant?
If so, should we create Issue and PR similar to the https://github.com/goalgorilla/open_social/pull/3751/ ?